sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -98,7 +98,8 @@ package object dsl {
       case _ => In(expr, list)
     }
 
-    def like(other: Expression): Expression = Like(expr, other)
+    def like(other: Expression, escapeCharOpt: Option[Char] = None): Expression =
```

Comment:
Please consider changing this to 
```
escapeChar: Char = '\\'
```
as well.

Reply:
OK